docs/gettingstarted.rst

```diff
@@ -61,7 +63,13 @@ You can also easily deploy your own REANA cloud instance by using the
    $ reana-cluster init
    $ reana-cluster status
    $ # set environment variables for reana-client
-   $ eval $(reana-cluster env)
+   $ eval $(reana-cluster env --incude-admin-token)
+
```

Comment:
Perhaps remove the warning and just leave a short CLI comment like:

```
eval $(reana-cluster env --include-admin-token) # if you are admin
```
Since we shall be explaining the token business elsewhere in corresponding guides.